src/main/kotlin/no/nav/familie/ba/soknad/api/services/pdl/PersonopplysningerService.kt

```diff
@@ -18,12 +18,13 @@ import org.springframework.stereotype.Service
 @Service
 class PersonopplysningerService(
     private val pdlClient: PdlBrukerClient,
-    private val pdlSystemClient: PdlSystemClient,
+    private val pdlApp2AppClient: PdlApp2AppClient,
     kodeverkClient: KodeverkClient
 ) {
+
     val kodeverkService = CachedKodeverkService(kodeverkClient)
     fun hentPersoninfo(personIdent: String, somSystem: Boolean = false): Person {
-        val response = if (somSystem) pdlSystemClient.hentPerson(personIdent) else pdlClient.hentPerson(personIdent)
+        val response = if (somSystem) pdlApp2AppClient.hentPerson(personIdent) else pdlClient.hentPerson(personIdent)
```

Comment:
(utenfor scope for PR)
Ser ikke helt behovet for å hente person som system her, slik det er nå så er `@PostMapping("/adressebeskyttelse")` helt åpen for alle 👀 

Dvs jeg kan sende inn et annet fnr til den entepunkten, som dessutom blir logget, her tenker jeg noe burde gjøres :)

Reply:
Bra catch, tror egentlig ikke vi trenger det endepunktet lenger heller, jeg skal slette det